problem4/problem4.js

```diff
@@ -6,7 +6,11 @@ var fahrenheitValue = document.getElementById('fahrenheit-value');
 function convert(celsiusValue) {
     console.log(celsiusValue);
 
-    fahrenheitValue.innerHTML = 'Your answer goes here';
+    fahrenheitValue.innerHTML = (celsiusValue*9/5)+32;
+
+    if (isNaN(celsiusValue)) {
+        return fahrenheitValue.innerHTML = 'Error!';
+    }
```

Comment:
This is okay, but a better way to do this would be to add the check for `isNaN` before the conversion calculation. That way, the function isn't doing extra work if it doesn't have to.

Reply:
Let's make the change, then once that's done and you see it working, go ahead and merge the change into the main branch! Good work! 👍